microlensing_photometry/astrometry/wcs.py

```diff
@@ -0,0 +1,93 @@
+from skimage.registration import phase_cross_correlation
+import numpy as np
+from astropy.coordinates import SkyCoord
+import scipy.spatial as sspa
+from skimage.measure import ransac
+from skimage import transform as tf
+from astropy.wcs import WCS,utils
+
+import microlensing_photometry.logistics.GaiaTools.GaiaCatalog as GC
+
+def find_images_shifts(reference,image,image_fraction =0.25, upsample_factor=1):
+    """
+    Estimate the shifts between two images. Generally a good idea to do a fraction in the middle,
```

Comment:
This comment needs to be a little more specific - I guess you mean a fraction of the overall image field of view.